src/families/ethereum/bridge/libcore.js

```diff
@@ -197,7 +198,7 @@ const estimateMaxSpendable = async ({
   const t = await prepareTransaction(mainAccount, {
     ...createTransaction(mainAccount),
     subAccountId: account.type === "Account" ? null : account.id,
-    recipient: "0x0000000000000000000000000000000000000000",
+    recipient: getAbandonSeedAddress(mainAccount.currency.id),
```

Comment:
Please undo, you moved 0x00..00 to abandon seed but it's not abandon seed. 0x00..00 can be used because it's both valid and proved that no one have the seed. We can't (easily) find similar for Bitcoin likes (actually we could derivate an address that is filled of 0 bytes - could be another idea but need coin formatters)